packages/docs/package.json

```diff
@@ -25,6 +26,8 @@
     "@storybook/web-components-vite": "^7.5.3",
     "@synergy-design-system/eslint-config-syn": "workspace:*",
     "@synergy-design-system/stylelint-config-syn": "workspace:*",
+    "@testing-library/react": "^14.1.0",
```

Comment:
What is this needed for?

Reply:
to create interactive stories, like the invalid one, where a user's action is mimicked